stl/src/atomic_wait.cpp

```diff
@@ -98,6 +98,8 @@ namespace {
 
 #if _ATOMIC_WAIT_ON_ADDRESS_STATICALLY_AVAILABLE
 
+#pragma comment(lib, "synchronization")

```

Comment:
I thought we had this comment, but it was replaced with linking using `CMakeList.txt`.
@BillyONeal , do you remeber anything relevant to this?

Reply:
The final binary which links against the `atomic_wait.obj` needs to also know to link in the `synchronization.lib`. It is insufficient to just link against the `synchronization.lib` at build time.

Reply:
> It is insufficient to just link against the `synchronization.lib` at build time.

Do I understand correctly that this isn't a problem for x86/x64 because `_ATOMIC_WAIT_ON_ADDRESS_STATICALLY_AVAILABLE` is zero because they have to support Windows 7?

Reply:
Yes, `_ATOMIC_WAIT_ON_ADDRESS_STATICALLY_AVAILABLE` is zero on x86/x64